packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -41,58 +41,60 @@ function storeMapper (stores) {
 class QuickTalkContainer extends React.Component {
   constructor () {
     super()
-    
+
     this.state = {
       comments: [],
       authors: {},
       authorRoles: {},
       postCommentStatus: asyncStates.initialized,
       postCommentStatusMessage: '',
       userId: '',
+      bearerToken: undefined,
     }
   }
-  
+
   componentDidMount () {
     if (!this.props.enabled) return
-    
+
     this.fetchComments()
     this.checkUser()
   }
-  
+
   componentDidUpdate (prevProps) {
     if (!this.props.enabled) return
-    
+
     const props = this.props
     if (props.subject !== prevProps.subject) {  // Note: this high level comparison actually works. Comparing props.subject?.id !== prevProps.subject?.id however causes a crash when getting a new subject, since prevProps.subject would have been removed from memory.
       this.fetchComments()
       this.checkUser()
     }
   }
-  
+
   /*
   Quick Fix: use authClient to check User resource within the QuickTalk component itself
   - see https://github.com/zooniverse/front-end-monorepo/discussions/2362
    */
   async checkUser () {
     const authClient = this.props?.authClient
     if (!authClient) return
-    
+
     const authorization = await getBearerToken(authClient)  // Check bearer token to ensure session hasn't timed out
     const user = await authClient.checkCurrent()
```

Comment:
I think these two calls are out of order. You should check the current user session before you can get a bearer token. Otherwise, I’m pretty sure you’ll find that the user is logged in but `authorization` is an empty string, and you can’t post comments.